includes/functions/functions_taxes.php

```diff
@@ -173,6 +173,12 @@ function zen_get_multiple_tax_rates($class_id, $country_id = -1, $zone_id = -1,
     if (is_array($rates_array)) {
         return $rates_array;
     }
+    // tax-exempt get no taxes
+    if (Customer::isTaxExempt() === true) {
+        return [
+            WHOLESALE_TAX_EXEMPT_DESCRIPTION => 0,
```

Comment:
Since `Customer::isTaxExempt` can return `true` for reasons other than a wholesale customer, please create another `Customer` class static method to return the customer's tax-exempt description, e.g.
```php
    public static function getTaxExemptDescription(): string
    {
        $tax_exempt_description = WHOLESALE_TAX_EXEMPT_DESCRIPTION;

        global $zco_notifier;
        $zco_notifier->notify('NOTIFY_CUSTOMER_TAX_EXEMPT_DESCRIPTION', [], $tax_exempt_description);

        return $tax_exempt_description ;
    }
```

... and then use that here (on line 178-180):
```php
        return [
            Customer::getTaxExemptDescription() => 0,
        ];
```

Reply:
no.  

i am not sure what other reasons a customer may be tax exempt.

feel free to put your own name on a function and notifier for which i see no use.

Reply:
FWIW, I'll point out that TaxJar makes a distinction between wholesale tax-exempt and government tax-exempt ... but I can live with the simple generalization of the language constant.